### PR TITLE
keylime: remove ALPHA tag

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

**Comment by dgdavid** [comment]
Thanks @aplanas!

Note for reviewers: related to https://github.com/yast/skelcd-control-MicroOS/issues/49

File: package/skelcd-control-MicroOS.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Jan 25 09:39:34 UTC 2022 - Alberto Planas Dominguez <aplanas@suse.com>
+
+- Remove ALPHA tag for the Keylime services
+- Reference the new place for the configuration file
```
Comment (dgdavid) [inline package/skelcd-control-MicroOS.changes:5]: A reference is needed to please our internal Jenkins.

Use `gh#yast/skelcd-control-MicroOS#49`

Comment (jreidinger) [reply package/skelcd-control-MicroOS.changes:5]: also version bump is needed, otherwise it won't be submited

Comment (lslezak) [reply package/skelcd-control-MicroOS.changes:5]: IIRC the accepted format is `gh#skelcd-control-MicroOS#49` ...

**Review by lslezak — COMMENTED** [review]
Also bump the version using `rake version:bump`

File: package/skelcd-control-MicroOS.changes (hunk 1 above)
Comment (aplanas) [reply package/skelcd-control-MicroOS.changes:5]: should be done (using @lslezak format)

**Comment by aplanas** [comment]
> Also bump the version using `rake version:bump`

should be done

**Review by jreidinger — APPROVED** [review]

File: package/skelcd-control-MicroOS.changes (hunk 1 above)
Comment (dgdavid) [reply package/skelcd-control-MicroOS.changes:5]: Just for the record, the long format is valid too. See https://github.com/openSUSE/packaging_rake_tasks/pull/54/files#diff-8045d97188605edfd44f8902c7fc3d70ad506f0cc0dc38f2b982f9dccf08e091R50


In fact, I used it yesterday at https://github.com/yast/skelcd-control-MicroOS/pull/48

**Pull request merged**